lib/index.d.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference types="node" />
+import { Transform } from 'stream';
+interface Options {
+    readableObjectMode: true;
+    highWaterMark: number;
+}
+interface Mark {
+    t: number;
+    i: number[];
+    v: number[];
+}
+export declare class Codegen extends Transform {
```

Comment:
I would love some feedback here. I'm sure I have some of the types wrong, especially any that're set to `any` such as the `fftData`.